react/src/pages/EnvironmentPage.tsx

```diff
@@ -50,12 +51,15 @@ const EnvironmentPage = () => {
     >
       <Flex
         style={{
-          display: curTabKey === 'image' ? 'block' : 'none',
-          paddingTop: token.paddingContentVerticalSM,
+          display: 'block',
         }}
       >
```

Comment:
This seems to be a remnant of code written to manage the display of the web component, how about removing this line?